how-to/customize-workspace/client/src/modules/endpoints/local-storage/platform-local-storage.ts

```diff
@@ -40,14 +40,14 @@ export class PlatformLocalStorage<T = unknown> implements PlatformStorage<T> {
 		}
 	}
 
-	public async getAll(query?: string): Promise<T[]> {
+	public async getAll(query?: string): Promise<{ [key: string]: T }> {
```

Comment:
Changing the response type I see you have updated the page workspace saving, but what about other spots localstorage is used, page bounds, integration preferences. Has that been impacted?